filebeat/module/nginx/access/_meta/fields.yml

```diff
@@ -3,128 +3,92 @@
   description: >
     Contains fields for the Nginx access logs.
   fields:
-    - name: remote_ip_list
-      type: array
+    - name: body_sent.bytes
+      type: long
+      format: bytes
       description: >
-        An array of remote IP addresses. It is a list because it is common to include, besides the client
-        IP address, IP addresses from headers like `X-Forwarded-For`. See also the `remote_ip` field.
+        The number of bytes of the server response body.
+
+    - name: remote_ip_list
+      type: alias
+      path: network.forwarded_ip
     - name: remote_ip
-      type: keyword
-      description: >
-        Client IP address. The first public IP address from the `remote_ip_list` array. If no public IP
```

Comment:
I wonder if at some stage we regret removing these descriptions. Ok for now, it's also in git.

Reply:
Can alias fields have a description? If so, I don't mind restoring the definitions.

There's a bunch of work needing to happen there anyway for doc generation, right? At this time, these aliases get the module's field name replaced with the ECS field name instead.

So if you want to keep the old field descriptions there, we'll need to have a doc generator that 

- lists the old name
- mentions that it's now an alias to the new field, with a mention of the new field (even a link to its definition)
- if a description is provided, provide the description of the deprecated field